packages/performance-tests/test/TestContext.ts

```diff
@@ -44,13 +44,16 @@ export async function *getTestIModels() {
 
     for await (const iModel of iModels) {
       const iModelId = iModel.id;
-      yield {
+      const iModelToCheck:TestIModel = {
         name: iModel.displayName,
         iModelId,
         iTwinId,
         tShirtSize: getTShirtSizeFromName(iModel.displayName),
         load: async () => downloadAndOpenBriefcase({ iModelId, iTwinId }),
-      };
+      }
+      if(filter(iModelToCheck)){
```

Comment:
make sure to run prettier + eslint, CI will fail on this spacing

Reply:
done